# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxUserFontSet.h
# Commit: ebecfbeb55c6
# Full Hash: ebecfbeb55c6fcbd730c2781cadfd850b26e4a26
# Author: Cameron McCormack <cam@mcc.id.au>
# Date: 2019-06-07 09:47:19
# Regressor Bug: 1490792
# File Overlap Count: 1
# Description:
#   Bug 1490792 - Part 2: Make LoadPlatformFont length argument not a reference. r=jfkthame
#   
#   We don't write to it.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D33895
# ==============================================================================

diff -r 478ac1179fa6 -r ebecfbeb55c6 gfx/thebes/gfxUserFontSet.h
--- a/gfx/thebes/gfxUserFontSet.h	Thu Jun 06 18:23:27 2019 +0000
+++ b/gfx/thebes/gfxUserFontSet.h	Thu Jun 06 18:23:29 2019 +0000
@@ -661,7 +661,7 @@
   // returns true if platform font creation successful
   // Ownership of aFontData is passed in here; the font must
   // ensure that it is eventually deleted with free().
-  bool LoadPlatformFont(const uint8_t* aFontData, uint32_t& aLength);
+  bool LoadPlatformFont(const uint8_t* aFontData, uint32_t aLength);
 
   // store metadata and src details for current src into aFontEntry
   void StoreUserFontData(gfxFontEntry* aFontEntry, bool aPrivate,
